src/index.ts

```diff
@@ -541,9 +557,11 @@ class RocPlayground {
     // Update UI first
     const editorContainer = document.getElementById("editorContainer");
     const replContainer = document.getElementById("replContainer");
+    const examplesSidebar = document.querySelector(".examples-sidebar") as HTMLElement;
```

Comment:
I moved the examples to a sidebar to achieve a familiar view just like in an IDE. It also makes them easier to discover, users may overlook them when they're all the way down in the footer.